draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1776,6 +1776,14 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
   <li><t>Under what conditions the field can be used; e.g., only in
     responses or requests, in all messages, only on responses to a
     particular request method, etc.</t></li>
+   <li><t>What the scope of applicability for the information conveyed in
+    the header is. By default, headers apply only to the message they are
```

Comment:
field?

Reply:
Fixed. It's going to take a while to do this consistently...